python/core/qgsmaprenderertask.sip

```diff
@@ -0,0 +1,34 @@
+class QgsMapRendererTask : QgsTask
+{
+%TypeHeaderCode
+#include <qgsmaprenderertask.h>
+%End
+
+  public:
+
+    enum ErrorType
+    {
+      ImageAllocationFail = 1,
+      ImageSaveFail
+    };
+
+    QgsMapRendererTask( const QgsMapSettings &ms,
+                        const QString &fileName,
+                        const QString &fileFormat = QStringLiteral( "PNG" ) );
+
+    QgsMapRendererTask( const QgsMapSettings &ms,
+                        QPainter *p );
+
+    void addAnnotations( QList< QgsAnnotation * > annotations );
```

Comment:
I _think_ it does handle lists.

I am however wondering whether it would not be nicer API if user of the API did not have to clone annotations - and cloning would be done inside addAnnotations()... then it would not even require /Transfer/